sklearn/preprocessing/_data.py

```diff
@@ -3034,7 +3048,8 @@ def _more_tags(self):
         return {'allow_nan': True}
 
 
-def power_transform(X, method='yeo-johnson', standardize=True, copy=True):
+@_deprecate_positional_args
+def power_transform(X, *, method='yeo-johnson', standardize=True, copy=True):
```

Comment:
You've not decorated the class version of this. I'm happy for methods to be positional.,..?